src/app/feed/feed-post-icon-row/feed-post-icon-row.component.html

```diff
@@ -1,152 +1,350 @@
-<div class="js-feed-post-icon-row__container fs-14px text-grey5 d-flex justify-content-between unselectable">
+<section>
   <div
-    class="cursor-pointer d-flex align-items-center"
-    (click)="openModal($event)"
-    data-toggle="modal"
-    data-target=".js-feed-post-icon-row__comment-modal"
+    *ngIf="postReactionCounts.Total > 0 && !hideSummary"
+    class="reactions-summary__container d-inline-flex text-secondary pt-10px"
+    (click)="openReactionsDetails($event)"
   >
-    <div>
-      <i-feather name="message-square" class="feed-post-icon-row__icon reply-icon"></i-feather>
-    </div>
-    <div>
-      <span *ngIf="!hideNumbers">{{ postContent.CommentCount }}</span>
+    <ng-container *ngFor="let reaction of postReactionCounts.Counts | keyvalue">
+      <div
+        *ngIf="reaction.value > 0 && allowedReactions.includes(reaction.key)"
+        class="reaction-summary__reaction-item mr-5px"
+      >
+        <reactions-image [reaction]="reaction.key" [size]="16"></reactions-image>
+      </div>
+    </ng-container>
+
+    <div class="reactions-summary__toggle-summary text-secondary">
+      {{ globalVars.abbreviateNumber(postReactionCounts.Total, 0) }}
+      {{ globalVars.pluralize(postReactionCounts.Total, "reaction") }}
     </div>
   </div>
 
-  <div
-    (click)="handleRepostClick($event)"
-    class="btn-group cursor-pointer d-flex align-items-center"
-    dropdown
-    #dropdown="bs-dropdown"
-    [ngClass]="{
-      'fc-green': postContent.PostEntryReaderState ? postContent.PostEntryReaderState.RepostedByReader : false
-    }"
-  >
-    <div>
-      <a
-        class="link--unstyled"
-        dropdownToggle
-        id="repostActionsButton"
-        data-toggle="dropdown"
-        aria-haspopup="true"
-        aria-expanded="false"
-      >
-        <i-feather name="repeat" class="feed-post-icon-row__icon repost-icon"></i-feather>
-      </a>
-    </div>
+  <div class="js-feed-post-icon-row__container fs-14px text-grey5 d-flex justify-content-between unselectable">
```

Comment:
You can ignore changes from `:22` to `:104`, because what actually happened is that I added reactions summary on top of this file, and added another parent wrapper. Because of this, Git shows those below as changed lines and some changes can look confusing (but actually nothing changed there)